cmd/merge.go

```diff
@@ -56,7 +56,7 @@ func MergeCommand(repo *core.Repo, gh func(context.Context) core.Gh) *cli.Comman
 			)
 			defer cancel()
 			if err := merger.Merge(mergeContext, pr); err != nil {
-				return cli.Exit("could not merge", 1)
+				return cli.Exit(fmt.Sprintf("could not merge. Err: %s", err.Error()), 1)
```

Comment:
I think it's not necessary to print this. We print in `PrintFailure` in the merger already. I don't think there are a lot of cases where it's going to be clear to the user what they need to do if we print the message ?

Reply:
ok, I did not have the `PrintFailure` in mind